e2e/setup-local-fdb-cluster.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Set up local FDB non-HA cluster on a 4-node kind k8s cluster
+
+SkipCreatingLocalCluster=${1:-0}
+
+cluster="local-cluster"
+
+if ${SkipCreatingLocalCluster} != 0; then
+    echo "===Start creating k8s cluster on kind"
+    kind create cluster --name ${cluster} --config ./local-cluster-config.yaml
+else
+    echo "===Skip creating k8s cluster on kind"
+    kind get clusters
+fi
+
+echo "===Start building operator"
+cd ..
+echo "---We should be at reop\'s root directory: "
+pwd
```

Comment:
Is there any need to print out the current path?

Reply:
It's just for user's information, letting them know which directory the command is running. I think it's easier for user to debug, in case script fails in the middle.